homeassistant/components/group/light.py

```diff
@@ -159,6 +162,14 @@ def should_poll(self) -> bool:
 
         if ATTR_COLOR_TEMP in kwargs:
             data[ATTR_COLOR_TEMP] = kwargs[ATTR_COLOR_TEMP]
+            for entity_id in list(self._entity_ids):
```

Comment:
Add a comment here what is going on . 

Also, why are you wrapping `self._entity_ids` in `list(…)` ? We should never mutate `self._entity_ids`

Reply:
We need a new list to mutate so we don't modify the old one. Addressed this in updated commit.